controllers/api.controllers.js

```diff
@@ -53,7 +53,16 @@ const postComment = (req, res, next) => {
     })
     .catch(next)
 }
+// Ticket 8
+const patchVotes = (req, res, next) => {
+    const changeVote = req.body.inc_votes
+    const articleId = req.params.article_id
+    updateVotes(articleId, changeVote)
+    .then((body) => {
+        res.status(200).send(body[0])
```

Comment:
I would probably send back rows[0] instead so we can send this back on a key of articles